<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnalysisWidget</class>
 <widget class="QWidget" name="AnalysisWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="updateHist">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Historgram</string>
   </property>
  </widget>
  <widget class="QLabel" name="startTimeLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Start time:</string>
   </property>
  </widget>
  <widget class="QLabel" name="endTimeLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>End Time:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="useCurrentTime">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Set end time to
 current time</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="convertButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Convert channel to energy</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>392</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="Equation">
    <item>
     <widget class="QLabel" name="E">
      <property name="text">
       <string>E =</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="c2">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Ch2">
      <property name="text">
       <string>*Ch^2  +</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="c1">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Ch">
      <property name="text">
       <string>*Ch  +</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="c0">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="fileNameInstruction">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Data File Name:
 (Must be .txt, 2 columns w/ time and energy)</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="dataFileName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;out_20120907__1805_17_471&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="startTime">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2012</year>
     <month>5</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="maximumDateTime">
    <datetime>
     <hour>23</hour>
     <minute>59</minute>
     <second>59</second>
     <year>7999</year>
     <month>12</month>
     <day>31</day>
    </datetime>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="endTime">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2012</year>
     <month>9</month>
     <day>1</day>
    </datetime>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="refTime">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2012</year>
     <month>5</month>
     <day>1</day>
    </datetime>
   </property>
  </widget>
  <widget class="QLabel" name="refTimeLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Reference time used in data file:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
